app/payment-link-v2/amount/get-index.controller.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+// Custom dependencies
+const { response } = require('../../utils/response')
+const { getSessionVariable } = require('../../utils/cookie')
+const paths = require('../../paths')
+const replaceParamsInPath = require('../../utils/replace-params-in-path')
+const { NotFoundError } = require('../../errors')
+
+module.exports = (req, res, next) => {
+  const product = req.product
+  const data = {
+    productExternalId: product.external_id,
+    productName: product.name,
+    productDescription: product.description,
+    productReferenceEnabled: product.reference_enabled,
+    captchaChallengeFailed: req.captchaChallengeFailed
```

Comment:
We're not using `productDescription` `productReferenceEnabled` or `productReferenceEnabled` in the view, so remove these.

Reply:
Done - thanks